App.js

```diff
@@ -7,108 +7,18 @@
  */
 
 import React from 'react';
-import {
-  SafeAreaView,
-  StyleSheet,
-  ScrollView,
-  View,
-  Text,
-  StatusBar,
-} from 'react-native';
 
-import {
-  Header,
-  LearnMoreLinks,
-  Colors,
-  DebugInstructions,
-  ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
+import {UserComponent} from './screens';
+import {SafeAreaView, ScrollView} from 'react-native';
 
-const App: () => React$Node = () => {
+export const App = () => {
   return (
     <>
```

Comment:
Instead of this, can we use `Fragment` rather? `Fragment` doesn't leave a footprint on the DOM so be aware when to use it.